validation.md

```diff
@@ -2241,6 +2244,84 @@ Occasionally, you may want to attach additional validation rules to your default
         // ...
     });
 
+<a name="validating-emails"></a>
+## Validating Emails
+
+Laravel provides a variety of validation rules that may be used to validate uploaded files, such as `rfc`, `strict` and `dns`. While you are free to specify these rules individually when validating emails, Laravel also offers a fluent file validation rule builder that you may find convenient:
+To ensure that emails are valid according to your application's requirements, you may use `Rule` class to fluently define the rule via ` Rule::email()`:
+
+```php
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Validation\Rule;
+
+$validator = Validator::make($request->all(), [
+    'email' => ['required', Rule::email()],
+]);
+```
+The Email rule object allows you to easily customize how emails are validated for your application, such as specifying that emails require RFC compliance, DNS checks, native PHP filtering, or spoof detection:
+
+```php
+// Basic RFC compliance...
+Rule::email()->rfcCompliant();
+
+// Strict RFC compliance...
+Rule::email()->rfcCompliant(strict: true);
+
+// Check for valid MX records...
+Rule::email()->validateMxRecord();
+
+// Prevent spoofing...
+Rule::email()->preventSpoofing();
+```
+Of course, you may chain all the methods in the examples above:
+
+```php
+Rule::email()
+    ->rfcCompliant(strict: true)
+    ->validateMxRecord()
+    ->preventSpoofing();
+```
+
+> [!WARNING]  
+> The `validateMxRecord()` and `preventSpoofing()` validators require the PHP `intl` extension.
+
+<a name="defining-default-email-rules"></a>
+
+Defining Default Email Rules
+You may find it convenient to specify the default validation rules for emails in a single location of your application. You can easily accomplish this using the Email::defaults method, which accepts a closure. The closure given to the defaults method should return the default configuration of the Email rule. Typically, the defaults rule should be called within the boot method of one of your application's service providers:
+
+```php
+use Illuminate\Validation\Rules\Email;
+
+/**
+ * Bootstrap any application services.
+ */
+public function boot(): void
+{
+    Email::defaults(function () {
+        $rule = (new Email())->rfcCompliant(strict: true)->preventSpoofing(),
+
+        return $this->app->isProduction()
+                    ? $rule->validateMxRecord()
+                    : $rule;
```

Comment:
That almost looks like it could use the [`Conditionable`](https://github.com/laravel/framework/blob/11.x/src/Illuminate/Conditionable/Traits/Conditionable.php) trait 🤔

Reply:
Quite a copy paste of the current Password rule documentation. The actual `Email` rule is already Conditionable. This piece of code is just an example, and using `when` probably feels bussier for new Laravel devs

Reply:
touché